pkg/core/localize/locales/en/cmd/kafka.en.toml

```
@@ -559,120 +559,120 @@ one = 'Format in which to display the Kafka topic (choose from: "json", "yml", "
 one = 'Format in which to display the Kafka topic (choose from: "json", "yml", "yaml")'
 
 [kafka.topic.produce.flag.key.description]
-one = 'The message key. Empty if not set'
+one = 'Message key. Empty if not set'
 
 [kafka.topic.produce.flag.key.file]
-one = 'Path to file containing message sent'
+one = 'Path to file containing sent message'
```

```suggestion
one = 'Path to file containing message value'
```

This is actually just the value for the message

---

Done